core-services/image-mirroring/openshift/mapping_origin_4_9

```diff
@@ -135,6 +135,7 @@ registry.ci.openshift.org/origin/4.9:jenkins quay.io/openshift/origin-jenkins:4.
 registry.ci.openshift.org/origin/4.9:jenkins-agent-base quay.io/openshift/origin-jenkins-agent-base:4.9 quay.io/openshift/origin-jenkins-agent-base:4.9.0 quay.io/openshift/origin-jenkins-agent-base:latest
 registry.ci.openshift.org/origin/4.9:jenkins-agent-maven quay.io/openshift/origin-jenkins-agent-maven:4.9 quay.io/openshift/origin-jenkins-agent-maven:4.9.0 quay.io/openshift/origin-jenkins-agent-maven:latest
 registry.ci.openshift.org/origin/4.9:jenkins-agent-nodejs quay.io/openshift/origin-jenkins-agent-nodejs:4.9 quay.io/openshift/origin-jenkins-agent-nodejs:4.9.0 quay.io/openshift/origin-jenkins-agent-nodejs:latest
+registry.ci.openshift.org/origin/4.9:jenkins-sync-plugin-scripts quay.io/openshift/origin-jenkins-sync-plugin-scripts:4.9 quay.io/openshift/origin-jenkins-sync-plugin-scripts:4.9.0 quay.io/openshift/origin-jenkins-sync-plugin-scripts:latest
```

Comment:
/assign @smarterclayton 

we are exposing e2e's in the jenkins sync plugin so that they can be consumed by the openshift/jenkins repo (i.e. the one for our openshift jenkins image

doing this based on @adambkaplan experience with step registry and the PR https://github.com/openshift/release/pull/20806 you approved for him last month

long long term ... we envision this as a step in the path to removing the jenkins image from the payload (via decoupling from the openshift/origin suites)